app/models/user.rb

```diff
@@ -638,8 +657,11 @@ def permissions
       can_create_groups: {
         scope: groups_with_create_access,
       },
-      can_read_groups: {
```

Comment:
Is the rest of the codebase updated to understand that `can_read_groups` does not exist anymore?

Reply:
I'm a bit worried because `has_permission?` in user.rb does not check whether the name of the permission is valid.
`some_random_user.has_permission?(:give_me_bananas)` will simply return `false` at the moment without complaining about the (intentionally) ridiculous and non-existent `:give_me_bananas` permission name.

This means that in the context of this PR, all old calls using `current_user.has_permission?(:can_read_groups)` will continue to "work", i.e. they will not throw an error but they will always return false even when the user has full (i.e. past) read access to the group.

Reply:
Yes these doesn't exist anymore. I made sure that.